### PR TITLE
review https://github.com/NixOS/nixpkgs/pull/239450

### PR DESCRIPTION
**Pull request by TheBrainScrambler**
Things changed:
- split name into pname/version. This is the recommended way to do it, don't know why you use name
- left the unwrapped binary too in case someone is unhappy with the wrapper or wants to make his own wrapper
- https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md: "meta.description must: [...] Not end with a period (or any punctuation for that matter)."
And some minor stuff

### CODE REVIEW

File: pkgs/games/fallout-ce/build.nix [hunk 1]
```diff
@@ -28,9 +29,11 @@ stdenv.mkDerivation (finalAttrs: {
 
   installPhase = ''
     runHook preInstall
-    install -D ${name} $out/bin/${name}
+
+    install -D ${name} $out/bin/${name}-unwrapped
     install -D ${./wrapper.sh} $out/bin/wrapper.sh
-    wrapProgram $out/bin/${name} --run $out/bin/wrapper.sh
+    wrapProgram $out/bin/${name}-unwrapped $out/bin/${name} --run $out/bin/wrapper.sh
```
Comment (hughobrien) [inline pkgs/games/fallout-ce/build.nix]:
```suggestion
    makeWrapper $out/bin/${pname}-unwrapped $out/bin/${pname} --run $out/bin/wrapper.sh
```

File: pkgs/games/fallout-ce/build.nix [hunk 2]
```diff
@@ -28,9 +29,11 @@ stdenv.mkDerivation (finalAttrs: {
 
   installPhase = ''
     runHook preInstall
-    install -D ${name} $out/bin/${name}
+
+    install -D ${name} $out/bin/${name}-unwrapped
```
Comment (hughobrien) [inline pkgs/games/fallout-ce/build.nix]:
```suggestion
    install -D ${pname} $out/bin/${pname}-unwrapped
```

File: pkgs/games/fallout-ce/fallout2-ce.nix [hunk 3]
```diff
@@ -4,18 +4,20 @@
...
 callPackage ./build.nix rec {
-  name = "fallout2-ce";
+  pname = "fallout2-ce";
+  version = "1.2.0";
 
   src = fetchFromGitHub {
     owner = "alexbatalov";
-    repo = "fallout2-ce";
-    rev = "v1.2.0";
+    repo = pname;
```
Comment (hughobrien) [inline pkgs/games/fallout-ce/fallout2-ce.nix]:
```suggestion
    repo = "fallout2-ce";
```

There's a whole hullaballoo about repo = pname https://github.com/nix-community/nixpkgs-lint/issues/21

File: pkgs/games/fallout-ce/build.nix (hunk 1 above)
Comment (hughobrien) [reply pkgs/games/fallout-ce/build.nix]: `wrapProgram` only takes one positional arg, I suspect you meant `makeWrapper` https://nixos.org/manual/nixpkgs/stable/#fun-makeWrapper

**Comment by hughobrien** [comment]
> split name into pname/version. This is the recommended way to do it, don't know why you use name

Version wasn't used in the build.nix stage despite being passed in so I rejiggered it. It's obviously harmless though so easier to stick to the recommendations I guess.

**Review by hughobrien — APPROVED** [review]
In the interest of expediency I'm going to commit the suggestions but please yell if you disagree. Thanks again for helping with this.

**Comment by TheBrainScrambler** [comment]
Ah sorry looks like I didn't try to build my review, thought I did it :/ but yes it's good that you fixed those mistakes.

> > split name into pname/version. This is the recommended way to do it, don't know why you use name
> 
> Version wasn't used in the build.nix stage despite being passed in so I rejiggered it. It's obviously harmless though so easier to stick to the recommendations I guess.

Uuh, but it is used ? Line 16: `inherit pname version src;` Since you kept it there's nothing to change, but just to tell you that this change wasn't just harmless but actually needed. If you specify only `name` instead of `pname`/`version` it's going to think that `name` is `pname`-`version` basically. And since you don't give any `version` but just give the `pname` as `name` it would cause problems.

**Pull request merged**